react/components/Tooltip/Portal.tsx

```diff
@@ -63,12 +58,8 @@ const Portal = (props: PropTypes.InferProps<typeof propTypes>, ref) => {
     }
   }, [onRendered, mountNode])
 
-  return mountNode ? ReactDOM.createPortal(children, mountNode) : mountNode
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Maybe you can resolve this by adding the condition `mountNode && mountNode instanceof HTMLElement`

Reply:
Solved! thanks